spec/factories.rb

```diff
@@ -26,14 +46,41 @@ def to_s
       json { "" }
     end
 
+    trait :with_id do
```

Comment:
I _do not_ like this trait name, and I'm open to suggestions.

The `response` and `schema` factories have corresponding `with_id` traits, so that once we come up with a better name, we can change them both in tandem.